src/NetMQ.Tests/ProxyTests.cs

```diff
@@ -1,29 +1,13 @@
 ﻿using System;
 using System.Threading.Tasks;
 using NetMQ.Sockets;
-using NUnit.Framework;
+using Xunit;
 
 namespace NetMQ.Tests
 {
-    [TestFixture]
-    public class ProxyTests
+    public class ProxyTests : IClassFixture<NetMQConfigFixture>
     {
-        [OneTimeSetUp]
-        public void SetUp()
```

Comment:
The `IClassFixture<NetMQConfigFixture>` interface causes xUnit to run `NetMQConfigFixture.ctor` first, then run all the tests, then run `NetMQConfigFixture.Dispose`. So `Cleanup(true)` runs at the both the start and end of the fixture.

I'll try running it at the start and end of every test instead.